rmldatatfbbox/rmldatatfbbox/utils/helpers.py

```diff
@@ -0,0 +1,115 @@
+import os
+import json
+import cv2
+from pathlib import Path
+import tensorflow as tf
+from object_detection.utils import dataset_util
+
+### FUNCTIONS ###
+def construct_all(image_ids: list, temp_dir: Path):
+    labeled_images = {}
+    label_to_int_dict = {}
+
+    for image_id in image_ids:
+        labeled_images[image_id] = construct(image_id, temp_dir, label_to_int_dict)
+    
+    return labeled_images, label_to_int_dict
+
+def construct(image_id: str, temp_dir: Path, label_to_int_dict: dict):
+    if temp_dir is None:
+        cwd = Path.cwd()
+        data_dir = cwd / 'data'
```

Comment:
This is mainly a change to the core, but we should get rid of any use of the CWD (i.e. `cwd/data` and `cwd/dataset`). These should instead use the RavenML cache in `~/.ravenML/imagesets` and `~/.ravenML/datasets`.

Reply:
Believe me I know. I left use of CWD in Jigsaw because it was so integrated throughout I couldn't get rid of it easily... for the love of god and all that is holy, Python devs, please stop using CWD